tdrs-backend/tdpservice/users/permissions.py

```diff
@@ -121,8 +138,8 @@ def has_permission(self, request, view):
         if has_permission and request.user.is_data_analyst:
             has_permission = is_own_stt(request, view)
 
-        # TODO: Add a conditional for Regional manager
-        # https://github.com/raft-tech/TANF-app/issues/1052
+        if request.user.groups.filter(name="OFA Regional Staff").exists():
+            has_permission = is_own_region(request, view)
```

Comment:
Additionally, if you combine this with the suggestions above for `is_own_stt` and `is_own_region` you could consolidate this to:
```
if has_permission and (request.user.is_data_analyst or request.user.is_regional_staff):
  requested_stt = get_requested_stt(request, view)
  perms_function = is_own_region if request.user.is_regional_staff else is_own_stt
  has_permission = perms_function(request.user, requested_stt)
```

Reply:
I didn't actually seem like we needed to add any logic to those spots, because of the way it gets called there, it looked to me like it was calling this method.